extensions/kubeapi/projects/list.go

```diff
@@ -0,0 +1,36 @@
+package projects
+
+import (
+	"context"
+
+	v3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	"github.com/rancher/shepherd/clients/rancher"
+	"github.com/rancher/shepherd/pkg/api/scheme"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ListProjects is a helper function that uses the dynamic client to list projects in a specific cluster.
+func ListProjects(client *rancher.Client, namespace string, listOpt metav1.ListOptions) (*v3.ProjectList, error) {
+	dynamicClient, err := client.GetDownStreamClusterClient(LocalCluster)
```

Comment:
Same as the above comment, should this be a local cluster?

Reply:
Project CRDs are available only on the local cluster and not on the downstream cluster.